CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Update to Buildpack API 0.10. As a side-effect, all launch processes are now wrapped in `bash -c` instead of using CNB's `direct = false` directive, which is no longer available. ([#205](https://github.com/heroku/procfile-cnb/pull/205))
```

Comment:
Should we mention that `.profile.d/` and `.profile` scripts will also no longer work?

(See write-up in https://github.com/heroku/procfile-cnb/pull/150#issuecomment-1531423521)

Reply:
Also that `lifecycle` 0.17.x or higher is now required:
https://github.com/buildpacks/lifecycle/blob/main/README.md#supported-apis

Reply:
I think perhaps the need for `.profile` is less with the built-in inline-buildpack support:
https://buildpacks.io/docs/app-developer-guide/using-inline-buildpacks/

There is also this ~official `.profile` buildpack:
https://registry.buildpacks.io/buildpacks/buildpacksio/profile

As such, maybe we can skip/defer thinking about this for now, and perhaps even not support it at all for CNB by default?